rust/operator-binary/src/backend/k8s_search.rs

```diff
@@ -133,16 +137,15 @@ fn build_label_selector_query(
     vol_selector: &SecretVolumeSelector,
     pod_info: LabelSelectorPodInfo,
 ) -> Result<String, Error> {
-    let mut label_selector =
-        BTreeMap::from([(LABEL_CLASS.to_string(), vol_selector.class.to_string())]);
+    let mut labels = BTreeMap::from([(LABEL_CLASS.to_string(), vol_selector.class.to_string())]);
```

Comment:
issue: We are not using any label validation here. We might want to use `Labels::new()` and `.insert(Label::try_from(...))`.

Reply:
Resolved in 48a8242

Reply:
Looking good!